client/src/pages/Landing.tsx

```diff
@@ -0,0 +1,16 @@
+import { Link } from "react-router-dom";
+
+export default function About() {
```

Comment:
```export default function Landing()``` instead of ```About()```?

Reply:
These would be fixed as soon as someone starts on the page